packages/mobile_app/lib/screens/registration/accept_terms.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:mutable_wrappers/mutable_wrappers.dart';
+import 'package:language/language.dart';
+import 'package:app/theme/app_colors.dart';
+
+class AcceptTermsRow extends StatefulWidget {
```

Comment:
Its fine for now, but we should consider changing it later
1. It should not be box constrained
2. The text after the checkbox should also change its state. Ideally, would be using somekind of label for the checkbox.

Reply:
Oh, need to play with it. Maybe GetsureDetector, will see.